packages/upgrade/src/commands/migrate.js

```diff
@@ -0,0 +1,172 @@
+/**
+ * Copyright IBM Corp. 2019, 2019
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import inquirer from 'inquirer';
+import { UpgradeError } from '../error';
+import { logger } from '../logger';
+import { Workspace, getAvailableWorkspaces } from '../workspace';
+
+export async function migrate(options, upgrades = []) {
+  logger.verbose('running migrate command with options: %o', options);
+
+  const { cwd } = options;
+  const migrations = upgrades
+    .filter((upgrade) => {
+      return upgrade.migrations && upgrade.migrations.length > 0;
+    })
+    .map((upgrade) => {
+      return upgrade.migrations;
+    })
+    .flat();
```

Comment:
```suggestion
    .flatMap((upgrade) => {
      return upgrade.migrations;
    });
```

I think this will work as an alternative (just looking at it real quick)

Reply:
@joshblack Good eye, thanks!